### PR TITLE
Tightened internal exception standard

### PR DESCRIPTION
**Pull request by XedinUnknown**
Aims to incorporate solution for #15.

### CODE REVIEW

File: src/InternalExceptionInterface.php [hunk 1]
```diff
@@ -2,11 +2,26 @@
...
 namespace Dhii\Exception;
 
+use Dhii\Exception\UnitTest\InternalExceptionInterfaceTest;
+use Exception as RootException;
+
 /**
  * Represents an exception that is not caused by consumer actions.
  *
+ * If this exception is thrown, it means that the cause of the problem
+ * is not relates to any parameter passed to the unit that was invoked.
```
Comment (mecha) [inline src/InternalExceptionInterface.php]: *related

**Review by mecha — CHANGES_REQUESTED** [review]
No test?

**Comment by XedinUnknown** [comment]
Test for what?

**Comment by mecha** [comment]
Disregard. For some reason, I expected to see a test for `InternalExceptionInterface`, not realizing that this PR _modifies_ and not _introduces_ the interface. #brainFart

**Comment by XedinUnknown** [comment]
@mecha, can I have this approved then please?

**Review by mecha — APPROVED** [review]

**Pull request merged**